docs/Authentication.md

```diff
@@ -49,6 +49,8 @@ const authProvider = {
     getIdentity: () => Promise.resolve(),
     // get the user permissions (optional)
     getPermissions: () => Promise.resolve(),
+    // get the user authorization to perform an action on a resource (optional)
```

Comment:
"get the user authorization" is confusing. Are we asking the user for his authorization?